pkg/api/v20240812preview/openshiftcluster_validatestatic.go

```diff
@@ -408,3 +423,56 @@ func (sv openShiftClusterStaticValidator) validateDelta(oc, current *OpenShiftCl
 
 	return nil
 }
+
+func (sv openShiftClusterStaticValidator) validatePlatformWorkloadIdentityProfile(path string, pwip *PlatformWorkloadIdentityProfile) error {
+	// PlatformWorkloadIdentityProfile being empty is acceptable
+	if pwip == nil {
+		return nil
+	}
+
+	// Validate the PlatformWorkloadIdentities
+	for n, p := range pwip.PlatformWorkloadIdentities {
```

Comment:
In addition to checking each `ResourceID`, you could check that each `OperatorName` is non-zero. The actual values will have to be checked during dynamic validation, but it won't hurt to do a quick check for empty strings here.

Reply:
Done!